frontend/src/routes.js

```diff
@@ -19,7 +19,7 @@ const ROUTES = {
             'shopByShopID': base + '/v1/shops/%b'
         },
         'post':{
-            'insertMerchant': base+'/v1/insert/merchant'
```

Comment:
No url changes needed for ShopController in routes?

Reply:
No, all the screens dependent on `ShopController`'s routes are yet to be added.